source/playbooks/common/parse_input.py

```diff
@@ -81,11 +81,10 @@ def _get_resource_id(self, parse_id_pattern, resource_index):
                 self.resource_id = identifier_match.group(resource_index)
             else:
                 exit(f'ERROR: Invalid resource Id {identifier_raw}')
-            return
 
     def _get_standard_info(self):
         match_finding_id = re.match(
-            r'^arn:(?:aws|aws-cn|aws-us-gov):securityhub:(?:[a-z]{2}(?:-gov)?-[a-z]+-\d):\d{12}:subscription/(.*?)/v/(\d+\.\d+\.\d+)/(.*)/finding/(?i:[0-9a-f]{8}-(?:[0-9a-f]{4}-){3}[0-9a-f]{12})$',
+            r'^arn:(?:aws|aws-cn|aws-us-gov):securityhub:(?:[a-z]{2}(?:-gov)?-[a-z]+-\d):\d{12}:'+'subscription/(.*?)/v/(\d+\.\d+\.\d+)/(.*)/finding/(?i:[0-9a-f]{8}-(?:[0-9a-f]{4}-){3}[0-9a-f]{12})$',
```

Comment:
Separating regex to cut down on regex complexity.

Reply:
this might make sonarqube happy but I don't think it really helps with the readability of the code. If we're going to try to fix this anyway, could this possibly be changed into named pieces so that I can more easily reason about what these regex's are at a glance?

for example:
```
arnPattern = '...'
findingPattern= '...'
match_finding_id = re.match(arnPattern + ":" + findingPattern)
```

Reply:
I'm not really sure that it improves readability, it's still a long regex. I did consider something along the lines of your code snippet, but ultimately didn't do it because there was no opportunity for re-usability on this file.